pypasta/Repository/Mbox.py

```diff
@@ -179,28 +179,51 @@ def load_file(filename, must_exist=True):
     return f
 
 
+def load_mbox_file(f_mbox_raw, name, d_mbox, mode='raw'):
+    if not os.path.exists(f_mbox_raw):
+        log.error('not a file or directory: %s' % f_mbox_raw)
+        quit(-1)
+
+    cwd = os.getcwd()
+    os.chdir(os.path.join(cwd, 'tools'))
+    ret = call(['./process_mailbox_maildir.sh', mode,
+                name, d_mbox, f_mbox_raw])
+    os.chdir(cwd)
+    if ret == 0:
+        log.info('  ↪ done')
+    else:
+        log.error('Mail processor failed!')
+
+
 class MailContainer:
     @staticmethod
-    def load_index(f_index):
+    def load_index(f_index, is_patchwork_index=False):
```

Comment:
TBH, I don't like this approach. We're calling the method like

load_index('/filename', True)

No one knows what "True" or "False" means, if we use it as an argument. And I'm sure we don't even need this argument.

Reply:
Since it's a keyword argument, the caller will have to include the keyword ie `is_patchwork_index` while making the call. Thus making it explicit.